product_hunt_utils.py

```diff
@@ -89,23 +271,57 @@ def follow_person_using_person_id(person_id: int)-> bool:
             'query': 'mutation UserFollowCreate($input:UserFollowCreateInput!){response:userFollowCreate(input:$input){node{id ...UserFollowButtonFragment __typename}__typename}}fragment UserFollowButtonFragment on User{id followersCount isFollowed __typename}',
         }
 
-        res = requests.post(PRODUCT_HUNT_GRAPHQL_API = 'https://www.producthunt.com/frontend/graphql', cookies=cookies, headers=headers, json=json_data)
+      res = requests.post(PRODUCT_HUNT_GRAPHQL_API, cookies=cookies, headers=headers, json=json_data)
+      res = res.json()
 
-        res = res.json()
+      res = res["data"]["response"]["node"]
+
+      if res == None:
+          return False
+      else:
+          return True
+
+def follow_now(df,count: int,filePath: str)->bool :

```

Comment:
`follow_now` does not make any sense and infact confusing as you already have a function `follow_a_person` and both depict to do the same thing. User should know what the function do just by reading its name. Rename this function from `follow_now` to `follow_list_of_ids`